src/test/java/org/jenkinsci/plugins/configfiles/buildwrapper/ConfigFileBuildWrapperTest.java

```diff
@@ -186,11 +186,11 @@ public void correctConfigMustBeActiveInDropdown() throws Exception {
         boolean foundSecond = false;
         for (DomElement htmlElement : option) {
             final HtmlOption htmlOption = (HtmlOption) htmlElement;
-            if (htmlOption.getValueAttribute().equals(activeConfig.id)) {
+            if (htmlOption.getValue().equals(activeConfig.id)) {
```

Comment:
This is not valid apparently. The method changed in `HtmlInput` not in `HtmlElement` based on https://www.htmlunit.org/migration.html.

```suggestion
            if (htmlOption.getValueAttribute().equals(activeConfig.id)) {
```